REQUIRE

```diff
@@ -1,3 +1,3 @@
-julia 0.7
-BinDeps
-MathProgBase 0.5 0.8
+julia 1.0
+Compat
```

Comment:
This might be true. It is used in [build.jl](https://github.com/SCIP-Interfaces/SCIP.jl/blob/7e4054b185b360ff428e0de1f7985906137ee57e/deps/build.jl#L20) but that was just copy-pasted from some other solver wrapper.